Headers/AppKit/NSStackView.h

```diff
@@ -0,0 +1,75 @@
+/* Definition of class NSStackView
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: 08-08-2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#ifndef _NSStackView_h_GNUSTEP_GUI_INCLUDE
+#define _NSStackView_h_GNUSTEP_GUI_INCLUDE
+
+#import <AppKit/NSView.h>
+
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_9, GS_API_LATEST)
+
+#if	defined(__cplusplus)
+extern "C" {
+#endif
+
+// Gravity
+enum {
+    NSStackViewGravityTop = 1,
+    NSStackViewGravityLeading = 1,
+    NSStackViewGravityCenter = 2,
+    NSStackViewGravityBottom = 3,
+    NSStackViewGravityTrailing = 3
+};
+typedef NSInteger NSStackViewGravity;
+
+// Distribution
+enum {
+    NSStackViewDistributionGravityAreas = -1,
+    NSStackViewDistributionFill = 0,
+    NSStackViewDistributionFillEqually,
+    NSStackViewDistributionFillProportionally,
+    NSStackViewDistributionEqualSpacing,
+    NSStackViewDistributionEqualCentering
+};
+typedef NSInteger NSStackViewDistribution;
+
+typedef float NSStackViewVisibilityPriority;
+static const NSStackViewVisibilityPriority NSStackViewVisibilityPriorityMustHold = 1000; 
```

Comment:
The last time you did add ".0" to all the floatCGFloat values to make the type clearer.

Reply:
Sure, I'll do that, sorry.